main/display/display.h

```diff
@@ -34,13 +34,15 @@ typedef struct
     uint16_t h;
 } wsg_t;
 
+struct display;
+
 typedef void (*pxSetFunc_t)(int16_t x, int16_t y, paletteColor_t px);
 typedef paletteColor_t (*pxGetFunc_t)(int16_t x, int16_t y);
 typedef paletteColor_t* (*pxFbGetFunc_t)(void);
 typedef void (*pxClearFunc_t)(void);
-typedef void (*drawDisplayFunc_t)(bool drawDiff);
+typedef void (*drawDisplayFunc_t)(bool drawDiff,void (*fnBackgroundDrawCallback)(struct display* disp, int16_t x, int16_t y, int16_t w, int16_t h, int16_t up, int16_t upNum ));
```

Comment:
Please typedef the new function pointer. It's used plenty of places.

Reply:
done